e2e/pages/SignOut.js

```diff
@@ -19,6 +19,7 @@ class SignOut extends Page {
      * method which do the sign out.
      * @returns the main signin page
      */
+    //TODO move function to new class Toolbar
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS1135)